changelog/22.0/22.0.0/summary.md

```diff
@@ -133,6 +134,11 @@ This is the last time this will be needed in the `8.0.x` series, as starting wit
 
 The base system now uses Debian Bookworm instead of Debian Bullseye for the `vitess/lite` images. This change was brought by [Pull Request #17552].
 
+### <a id="key-range-vtorc"/>KeyRanges in `--clusters_to_watch` in VTOrc</a>
+VTOrc now supports specifying KeyRanges in the `--clusters_to_watch` flag. This is useful in scenarios where you don't need to restart a VTOrc instance if you run a reshard.
+For example, if a VTOrc is configured to watch `ks/-80`, then it would watch all the shards that fall under the KeyRange `-80`. If a reshard is run and, `-80` is split into new shards `-40`, and `40-80`, the VTOrc instance will automatically start watching the new shard without needing a restart.
```

Comment:
@GuptaManan100 I think this explains the "new" behaviour very well, but I feel like a user of the old way won't fully understand the risks

For example, a user with `--clusters_to_watch foo/-80` in the previous logic will watch 1 (or 0) shards only. After this change they would watch any shard from `-` to `80`, which could be surprising. So I guess I'm saying explaining the difference between the old logic and this PR might make this more clear

Reply:
What should I add to make it more clear? Should I add a line something like - 
`In the previous logic, watching a shard -80 would watch 1 (or 0) shard only. In the new system, since we interpret -80 as a key range, it can lead to a watch on multiple shards as described in the example.`

Reply:
@GuptaManan100 that extra detail is great 👍